### DISCUSSION TITLE
Build failing for 14.0 on Python 13.0 with Debian Bookworm

### DISCUSSION THREAD

**Discussion by daveisfera** [PyAV-Org/PyAV#1667] [4. Bugs]
## Overview

Tried upgrading to 14.0 and build failed

## Expected behavior

Build works and successfully passes our tests like previous builds

## Actual behavior

Fails trying to include file

Install command:
```
pip3 install av==14.0.0 --no-binary av
```

Error:
```
gcc -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O3 -Wall -fPIC -Iav/filter -I/usr/src/venv/include -I/usr/local/include/python3.13 -c src/av/filter/loudnorm.c -o build/temp.linux-x86_64-cpython-313/src/av/filter/loudnorm.o
       src/av/filter/loudnorm.c:1289:10: fatal error: loudnorm_impl.h: No such file or directory
        1289 | #include "loudnorm_impl.h"
```

## Investigation

Appears that this file should be included from the code for `av`

## Reproduction

Run `pip3 install` on Debian Bookworm with Python 3.13.0

## Versions

- OS: Debian Bookworm
- PyAV runtime:
Installing with `pip3 install` so not sure how to get this
- PyAV build:
Installing with `pip3 install` so not sure how to get this

## Research

I have done the following:

- [X] Checked the [PyAV documentation](https://pyav.basswood-io.com)
- [X] Searched on [Google](https://www.google.com/search?q=pyav+how+do+I+foo)
- [X] Searched on [Stack Overflow](https://stackoverflow.com/search?q=pyav)
- [X] Looked through [old GitHub issues](https://github.com/PyAV-Org/PyAV/issues?&q=is%3Aissue)

**Comment by WyattBlue** [comment]
9560bb8674e3f3e0d164c5c667e0064a0a057d7c should already fix this.

**Reply by daveisfera** [reply]
Great to hear! Any chance a 14.0.1 release could be rolled out to include this?

**Reply by WyattBlue** [reply]
I do plan to release a "14.0.1"
